data/pipeline.yaml

```
@@ -338,6 +391,67 @@ spec:
       secret: {secretName: quay}
     - name: ssh-key
       secret: {defaultMode: 384, secretName: ssh-key}
+  - name: to-args
+    container:
+      args: [--experiment-result, '{{inputs.parameters.katib-experiment-params}}',
+        '----output-paths', /tmp/outputs/Output/data]
```

THis does not look right:
```suggestion
        '--output-paths', /tmp/outputs/Output/data]
```

---

Not exactly sure why this happens...I guess this could be a kfp bug as it generates this code.